fastlane/swift/main.swift

```diff
@@ -38,7 +38,7 @@ class MainProcess {
 let process = MainProcess()
 process.startFastlaneThread()
 
-while !process.doneRunningLane, RunLoop.current.run(mode: RunLoopMode.defaultRunLoopMode, before: Date(timeIntervalSinceNow: 2)) {
+while !process.doneRunningLane, RunLoop.current.run(mode: RunLoop.Mode.default, before: Date(timeIntervalSinceNow: 2)) {
```

Comment:
But we're upgrading to Swift 5 in this PR 🤔 ?

Reply:
Nevermind, just read your comments below.
Can I understand why the desire to keep it compatible with Swift 4? Is it so users are able to use Fastlane.Swift on older versions of Xcode?

Reply:
That's the main reason, I mean, I could've bumped to Swift 5 a time ago but It seemed correct to me not to introduce  a breaking change like this without knowing if people would be affected for that. Also, the initial implementation was written in Swift 4 so I respected that.

Reply:
Makes sense!

Reply:
Just gonna leave this here: Swift 5 toolchain was first introduced in Xcode 10.2, which was officially released on March 25 2019
Perhaps it wouldn't be much of an issue in practice, but I see your point @minuscorp 👍 

We should probably include info like this in fastlane's analytics gathering (including how many people (or %) actually use Fastlane.Swift 🤔 )

Reply:
I see your point. I can change it back to swift 4.
But I think it's a good time to do it, swift 5 was introduced 2019/03, at it seems it will have a longer support cycle.
I sincerely don't see the point in preserving backwards compatibility to swift 4 for fastlane, which is usually used to deliver to the App Store, which enforced you to use Xcode 12 and iOS 14 SDK et al.
Besides upgrading swift 4 to swift 5 is not usually painful.

Reply:
Not that it is painful or that I'm not aware how deprecated it is, just saying that I'd prefer to first solve this issue in Swift 4 without any breaking changes and do a later PR to upgrade to Swift 5.
I think it is better to do one thing at a time, because this is not a complete migration of everything and I'd prefer to take a deeper look into it.